examples/next-14-app-router-ssr/public/_live-preview.ts

```diff
@@ -9,7 +9,7 @@ ContentfulLivePreview.init({
 ContentfulLivePreview.subscribe('save', {
   callback: async () => {
     const pathname = window.location.pathname;
-
-    return fetch(`/api/revalidate?pathname=${pathname}`);
+    await fetch(`/api/revalidate?pathname=${pathname}`);
+    window.location.reload();
```

Comment:
Hey! I am struggling to understand the difference between this and the next 13 example that we reload as well?

https://github.com/contentful/live-preview/blob/main/examples/next-13-app-router-ssr/public/_live-preview.ts

Also do you mind updating the README in the examples repo to explain what the difference is now between this and the next 13 example?

Reply:
Hello! the main difference now is just the route cache invalidation request which also may or may not be needed, I was showing this to @chrishelgert and he mentioned to keep it in.

I guess when this example was originally made it assumed the redirect within the route was working. Maybe it makes sense to consolidate these examples given how close they are now?

Reply:
just this https://github.com/contentful/live-preview/blob/main/examples/next-14-app-router-ssr/app/api/revalidate/route.ts

Reply:
Yes I think having one example for SSR would be better, if they do the same thing either way :)